package.json

```diff
@@ -62,14 +65,12 @@
     {
       "path": "lib/**/*([a-zA-Z-_]).js",
       "limit": "140KB",
-      "name": "components",
-      "webpack": false
```

Comment:
Don't need/want the `webpack` option, since we're checking the size of our own build command's output (vs using webpack to build).